src/components/__tests__/Mount.js

```diff
@@ -16,7 +16,7 @@ it('create, navigate, and edit thoughts', async () => {
   await editable.simulate('change', { target: { value: 'a' } })
 
   // create subthought
-  windowEvent('keydown', { key: 'Enter', ctrlKey: true })
+  windowEvent('keydown', { key: 'Enter', altKey: true })
```

Comment:
All `Ctrl + Enter` and related shortcuts for creating new thoughts should use the original `ctrlKey`. Sorry I forgot about those.